docs/src/modules/components/ChangeLog.js

```diff
@@ -317,7 +317,7 @@ export default function ChangeLog() {
           }}
         >
           {entries.map((entry) => (
-            <TimelineItem>
+            <TimelineItem key={entry.date.toISOString()}>
```

Comment:
Here is the key fix

I assume you did the file migration before the to `modules/components` before the merge so git did not understood the link between them

Reply:
Weird thing is that I actually did it _before_ 😅 Anyway, thank you for chiming ing! 🙌